### ISSUE TITLE
Visual Design:  Spaces list view

### ISSUE THREAD

**Issue by catrobson** [fabric8-ui/fabric8-ux#623]
Once stakeholder review is complete, finalize visual design pass on the spaces list page before we move into development.  Either coordinate with June on updates needed, or ask for Sketch file to do a visual design pass on top of her wireframes since they are close.

https://redhat.invisionapp.com/share/6XD26VFZB

**Comment by bdellasc** [comment]
Started on a few ideas around the visual indication for activity on a space. I added them to June's existing wireframe InVision Project: [OSIO-Space](https://redhat.invisionapp.com/share/6XD26VFZB#/253278949_ActivityIndicatorIdeas)

**Comment by bdellasc** [comment]
Reviewed the work with Stakeholders for #681, need to create a new story that fields/integrates the feedback on the design.

**Comment by bdellasc** [comment]
Held meeting on 9/29/17 with: Sj, Laura, Joshua, Aslak, Stevan, Todd, Corinne, Xavier, Max, Len
